contracts/oracle/Cargo.toml

```diff
@@ -19,3 +19,5 @@ sbt = { path = "../sbt" }
 
 [dev-dependencies]
 rand = "^0.7"
+base64 = "0.13"
+near-primitives = "0.1.0-pre.1"
```

Comment:
this is a very old version. Let's use https://crates.io/crates/near-primitives/versions

Reply:
DONE